src/toil/provisioners/azure/azureProvisioner.py

```diff
@@ -30,6 +30,7 @@
 from azure.common.credentials import ServicePrincipalCredentials
 from azure.mgmt.compute import ComputeManagementClient
 from azure.mgmt.network import NetworkManagementClient
+from toil.provisioners.azure import getAlternativeAzureZone
```

Comment:
I'm pretty sure you meant `from toil.provisioners.azure import getAzureZone` here.